great-idea/services.html

```diff
@@ -1,87 +1,94 @@
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+                <div class="blurb">
+                    <h2>UX / UI</h2>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+                <div class="blurb">
+                    <h2>Digital Marketing</h2>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+                <div class="blurb">
+                    <h2>Web Development</h2>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+                <div class="blurb">
+                    <h2>iOS Development</h2>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+                <div class="blurb">
+                    <h2>Android Development</h2>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
+                    <a class="button">Learn More</a>
+                </div>
+            </section>
+            <footer>
```

Comment:
ONE footer per page

Reply:
The Index and Services pages share a template, consisting of navigation (header), individual page content, and copyright (footer). Each page's content then has it's own similar 3-part structure within main element. The page content is then inserted into the theme template. Is this an acceptable structure?